README.md

```diff
@@ -1,72 +1,44 @@
-# OpenTelemetry-Go-Compile-Instrumentation
+# OpenTelemetry Go Compile Instrumentation
 
-OpenTelemetry-Go-Compile-Instrumentation provides compile time [OpenTelemetry](https://opentelemetry.io/) instrumentation for [Golang](https://golang.org/).
+> [!IMPORTANT]
+> This is a work in progress and not ready for production use. 🚨
 
-## Getting Started
-
-### Prerequisites
-
-- Go 1.23.0 or later
-- Git
-
-### Quick Start
-
-1. **Clone the repository:**
-   ```bash
-   git clone https://github.com/open-telemetry/opentelemetry-go-compile-instrumentation.git
-   cd opentelemetry-go-compile-instrumentation
-   ```
-
-2. **Build the instrumentation tool:**
-   ```bash
-   make build
-   ```
-
-3. **Run the demo with instrumentation:**
-   ```bash
-   make demo
-   ```
-
-   This will:
-   - Build the instrumentation tool (`otel` binary)
-   - Run the demo with compile-time instrumentation
-   - Execute the instrumented binary
-
-## How It Works
-
-The instrumentation tool injects OpenTelemetry code at compile time using Go's `-toolexec` flag. The process works as follows:
+This project provides a tool to automatically instrument Go applications with [OpenTelemetry](https://opentelemetry.io/) at compile time. The tool modifies the Go build process to inject OpenTelemetry code into the application without requiring manual changes to the source code.
 
-1. **Setup Phase**: Creates `.otel-build` directory and initializes logging
-2. **Go Build Phase**: Intercepts the `go build` command and adds `-toolexec=./otel` flag
-3. **Toolexec Phase**: The Go toolchain invokes the tool for each compilation step, allowing code injection. This phase is not visible to the user.
-
-### Logging
-
-The tool provides detailed logging to help debug instrumentation issues:
-- Logs are written to `.otel-build/debug.log` for setup and go actions
-- Toolexec action logs to stdout for immediate feedback
-- All logs include structured information about the build process
-
-### Inspecting the Instrumentation
-
-To see the result of instrumentation, navigate to the WORK directory that appears in the build output:
+## Getting Started
 
+1. Build the otel tool
 ```bash
-# Example WORK directory
-cd /var/folders/x9/fddsvlt5363c0plvvw8_2mr80000gn/T/go-build2020695287
-
-# Locate the main package under the b001 subdirectory
-ls -l b001
+$ git clone https://github.com/open-telemetry/opentelemetry-go-compile-instrumentation.git
+$ cd opentelemetry-go-compile-instrumentation
+$ make build
+```
 
-# Inspect the modified files
-cat modified.go
+2. Build the application with the tool and run it
+```bash
+$ make demo
 ```
 
+## Project Structure
+- `docs/`: Documentation for the project
```

Comment:
Just a suggestion. Seems that this duplicates information of 'docs/[api-design-and-project-structure.md](https://github.com/open-telemetry/opentelemetry-go-compile-instrumentation/blob/main/_docs/api-design-and-project-structure.md)
Maybe just link to this file:
```suggestion
See the [project structure](https://github.com/open-telemetry/opentelemetry-go-compile-instrumentation/pull/37/docs/api-design-and-project-structure.md).
```